starboard/shared/ffmpeg/ffmpeg_dispatch.h

```diff
@@ -105,6 +115,40 @@ class FFMPEGDispatch {
   unsigned (*avformat_version)(void);
   void (*av_register_all)(void);
 
+  void (*av_free)(void* ptr);
```

Comment:
This set of changes wasn't in the other branches, why was it added for this branch?

Reply:
I think it's because Thor ended up submitting another fix in trunk after the initial fix ( the gcc-6.3 and clang-3.9 were broken ) which made that change on top. I think it got squashed together in the cherry-pick. Either way lgtm Thor if builds pass now - but please respond to Jelle if i'm correct in the guess.

Reply:
no, these aren't due to the second cherrypick. Many of the functions initialized in starboard/shared/ffmpeg/ffmpeg_dynamic_load_dispatch_impl.cc from this initial change, weren't actually defined in ffmpeg_dispatch.h in 21.lts. so I had to add the symbols for the cherrypick to compile.